labware-library/src/labware-creator/labwareDefToFields.ts

```diff
@@ -10,6 +10,17 @@ import type { BooleanString, LabwareFields } from './fields'
 // NOTE: this is just String() with some typing for flow
 const boolToBoolString = (b: boolean): BooleanString => (b ? 'true' : 'false')
 
+export function getStackedLabwareZDimension(
+  def: LabwareDefinition2
+): number | undefined {
```

Comment:
```suggestion
): number | null {
```
Generally for our internal utils, we should return `null` instead of `undefined`. `undefined` generally represents a none value type typically returned from an external API.